src/main/resources/db/migration/V1_10__static_table_inserts.sql

```diff
@@ -0,0 +1,343 @@
+insert into court_location_category(
+    court_location_category_id,
+    court_location_category,
+    welsh_court_location_category,
+    created_time
+    )
+values(
+    '1',
+    'Admin Court',
+    ' ',
```

Comment:
we're using `' '` because the field cannot be null?

Reply:
Yes @ishratt - this is what @kavithagade12 asked me to do:

> _"non-mandatory column keept it blank"_

Reply:
All the columns in the Data model which starts with welsh are blank. and they are optional(non mandatory).

Reply:
Thanks for the comments, @kavithagade12 please correct me if I'm wrong here.. according to the flyway script `V1_9__create_tables.sql` this is what we have in the `court_location_category` table: `welsh_court_location_category varchar(128) NOT NULL` doesn't this mean this field cannot be null, hence mandatory? All the other columns starting with welsh seem to be optional as you said except for this one.

Reply:
As mentioned all the  fields which starts with welsh_*.* are non mandatory. Please change  welsh_court_location_category to nullable field.

Reply:
The following value should be null rather than '  ',